package.json

```diff
@@ -39,5 +37,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@types/jest": "^24.9.1",
```

Comment:
Thanks for looking into the dependencies and fixing the testing suites.

Reply:
What a headache. Glad ya got it working!